engine/src/execution_graph/logic_controllers/CacheMachine.h

```diff
@@ -1165,11 +1165,11 @@ class CacheMachine {
 	/// This variable is to keep track of if anything has been added to the cache. Its useful to keep from adding empty tables to the cache, where we might want an empty table at least to know the schema
 	bool something_added;
 	std::shared_ptr<Context> ctx;
-	std::shared_ptr<spdlog::logger> logger;
-	std::shared_ptr<spdlog::logger> cache_events_logger;
 	const std::size_t cache_id;
 	int cache_level_override;
 	std::string cache_machine_name;
+    std::shared_ptr<spdlog::logger> logger;
```

Comment:
remove ` std::shared_ptr<spdlog::logger> logger;` we no longer use it in here. Please also remove it from the constructor